init.sh

```diff
@@ -39,6 +42,23 @@ echo "------"
 echo resolver $(awk 'BEGIN{ORS=" "} /nameserver/{print $2}' /etc/resolv.conf | sed "s/ $/;/g") > /etc/api-gateway/conf.d/includes/resolvers.conf
 echo "   ...  with dns $(cat /etc/api-gateway/conf.d/includes/resolvers.conf)"
 
+sync_cmd="echo checking for changes ..."
+if [[ -n "${remote_config}" ]]; then
+    echo "   ... using a remote config from: ${remote_config}"
+    if [[ ${remote_config} == s3://* ]]; then
+      sync_cmd="aws s3 sync --exclude *resolvers.conf --exclude *environment.conf.d/*upstreams.http.conf --delete ${remote_config} /etc/api-gateway/"
+      echo "   ... syncing from s3 using command ${sync_cmd}"
+    else
+      echo "   ... but this REMOTE_CONFIG is not supported "
+    fi
+fi
+api-gateway-config-supervisor \
+        --reload-cmd="api-gateway -s reload" \
+        --sync-folder=/etc/api-gateway \
```

Comment:
Thanks @clehene . We could also expose the `sync-interval` as an env variable for docker. WDYT ?